nodeJS/express/introduction_to_express.md

```diff
@@ -0,0 +1,98 @@
+### Introduction
+
+In the previous lesson, you got up and running with Node. You learned how to read and write data from files on the server, and use those to serve up a multi-page website. You may have found this a somewhat verbose and cumbersome process and may have even wondered how on Earth we would manage more complex procedures.
+
+In this course, we are going to be using a backend framework called [Express](https://expressjs.com/). It will handle some of the more cumbersome things for us, and make the things we did previously far simpler. Express itself is an intentionally barebones and unopinionated framework; it allows us to do many things how we want, and to extend it with only the features we need. While this gives us great flexibility in how we do things, it can be a little tricky deciding between multiple viable options at times.
```

Comment:
Wording suggestion: 
```suggestion
In this course, we are going to be using a backend framework called [Express](https://expressjs.com/). It exposes an easy to use API that handles a lot of the implementation details for us. Express itself is an intentionally barebones and unopinionated framework; it allows us to do many things how we want, and to extend it with only the features we need. While this gives us great flexibility in how we do things, it can be a little tricky deciding between multiple viable options at times.
```

Reply:
Like the more streamlined wording. I've gone for even more streamlined as I'm not sure about explicitly saying "this is easy to use/has an easy to use XYZ", but the latter part is definitely more than sufficient.